<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsGeorefConfigDialogBase</class>
 <widget class="QDialog" name="QgsGeorefConfigDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Georeferencer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="mPointTipGroupBox">
     <property name="title">
      <string>Point tip</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="mShowIDsCheckBox">
        <property name="text">
         <string>Show IDs</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="mShowCoordsCheckBox">
        <property name="text">
         <string>Show coordinates</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="mResidualUnitsGroupBox">
     <property name="title">
      <string>Residual units</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QRadioButton" name="mPixelsButton">
        <property name="text">
         <string>Pixels</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="mMapUnitsButton">
        <property name="text">
         <string>Use map units if possible</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="mPdfReportGroupBox">
     <property name="title">
      <string>PDF report</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Left margin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QgsDoubleSpinBox" name="mLeftMarginSpinBox">
          <property name="prefix">
           <string/>
          </property>
          <property name="suffix">
           <string> mm</string>
          </property>
          <property name="showClearButton" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Right margin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QgsDoubleSpinBox" name="mRightMarginSpinBox">
          <property name="suffix">
           <string> mm</string>
          </property>
          <property name="showClearButton" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="mShowDockedCheckBox">
     <property name="text">
      <string>Show Georeferencer window docked</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>PDF map</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Paper size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="mPaperSizeComboBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">qgsdoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mShowIDsCheckBox</tabstop>
  <tabstop>mShowCoordsCheckBox</tabstop>
  <tabstop>mPixelsButton</tabstop>
  <tabstop>mMapUnitsButton</tabstop>
  <tabstop>mLeftMarginSpinBox</tabstop>
  <tabstop>mRightMarginSpinBox</tabstop>
  <tabstop>mPaperSizeComboBox</tabstop>
  <tabstop>mShowDockedCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
